bulldozer/config_v0.go

```diff
@@ -23,9 +23,10 @@ const (
 )
 
 type ConfigV0 struct {
-	Mode             ModeV0      `yaml:"mode"`
-	Strategy         MergeMethod `yaml:"strategy"`
-	DeleteAfterMerge bool        `yaml:"deleteAfterMerge"`
+	Mode                   ModeV0      `yaml:"mode"`
+	Strategy               MergeMethod `yaml:"strategy"`
+	DeleteAfterMerge       bool        `yaml:"deleteAfterMerge"`
+	AllowMergeWithNoChecks bool        `yaml:"allow_merge_with_no_checks"`
```

Comment:
The v0 configuration is deprecated and shouldn't get any new features. People who wish to configure this option should move to the supported v1 configuration.